service/src/main/kotlin/fi/espoo/evaka/pairing/MobileDeviceQueries.kt

```diff
@@ -0,0 +1,68 @@
+package fi.espoo.evaka.pairing
+
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.domain.NotFound
+import org.jdbi.v3.core.kotlin.mapTo
+import java.util.UUID
+
+fun insertDeviceData(tx: Database.Transaction, unitId: UUID): UUID {
```

Comment:
Didn't we decide to use the form `fun Database.Transaction.insertDeviceData(...)` at some point? Once again we're adding new code with two different styles :slightly_frowning_face:

Reply:
changed these too